src/masternodeman.cpp

```diff
@@ -811,8 +811,13 @@ std::vector<std::pair<int, CMasternode>> CMasternodeMan::GetMasternodeRanks(int
     BOOST_FOREACH (CMasternode &mn, vMasternodes)
     {
 
-        if (mn.nProtocolVersion < nMinProtocol || !mn.IsEnabled())
+        if (mn.nProtocolVersion < nMinProtocol)
             continue;
+        
+        if (!mn.IsEnabled()) {
```

Comment:
Undocumented in PR#68

Please either:
1. Document change in PR
2. Remove from PR